### ISSUE TITLE
Miscellaneous UI improvements

### ISSUE THREAD

**Issue by andrewgormley** [netbox-community/netbox#16853]
### NetBox version

v4.1.0-dev

### Feature type

Change to existing functionality

### Proposed functionality

- [x] (Misc) Edit icon/combo box missing alt text [Multiple]
- [x] (Racks) 'images and labels' button label is ambiguous [Multiple]
- [ ] (Buttons) On rack elevation page front and rear toggle button missing focus state on active part [Multiple]
- [x] (Misc) user-scalable="no" in the meta tag [Global]
- [ ] (User profile) Redundant link as both id and date link to the same place [Single]

### Use case

Fixing web accessibility issues ensures NetBox is best placed to serve users using screen readers and other accessibility tools.
